app/classifier/tasks/combo/index.cjsx

```diff
@@ -116,7 +116,7 @@ ComboTask = createReactClass
             task={taskDescription}
             workflowID={@props.workflow.id}
           >
-            <TaskComponent {...@props} autoFocus={i is 0} task={taskDescription} annotation={annotation} onChange={@handleChange.bind this, i} />
+            <TaskComponent {...@props} autoFocus={i is 0} task={taskDescription} taskKey={task} annotation={annotation} onChange={@handleChange.bind this, i} />
```

Comment:
Noting in `taskKey={task}` that `task` is an item from the combo task `tasks` property, per the `map` on line 100, which is a string of the task ID for a task in the combo task. 👍

Reply:
PFE's variable naming was not always consistent either.